libraries/bsg_manycore_loader.cpp

```diff
@@ -822,18 +775,7 @@ static int hb_mc_loader_column_validate_victim_cache(hb_mc_manycore_t *mc,
 static int hb_mc_loader_columns_validate_victim_cache(hb_mc_manycore_t *mc,
                                                       const hb_mc_eva_map_t *map)
 {
-        const hb_mc_config_t *cfg = hb_mc_manycore_get_config(mc);
-        hb_mc_idx_t n_columns = hb_mc_dimension_get_x(hb_mc_config_get_dimension_vcore(cfg));
-        int rc;
-
-        /* for each column... */
-        for (hb_mc_idx_t col = 0; col < n_columns; col++) {
-                rc = hb_mc_loader_column_validate_victim_cache(mc, map, col);
-                if (rc != HB_MC_SUCCESS)
-                        return rc;
-        }
-
-        return HB_MC_SUCCESS;
+	return hb_mc_manycore_validate_vcache(mc);
```

Comment:
Something's funky with the whitespace here

Reply:
I 'll fix all the whitespace.